daemon/execdriver/native/create.go

```diff
@@ -222,3 +227,17 @@ func (d *driver) setupLabels(container *libcontainer.Config, c *execdriver.Comma
 
 	return nil
 }
+
+func (d *driver) setupAdditionalGroups(container *libcontainer.Config, c *execdriver.Command) error {
+	container.AdditionalGroups = make([]int, len(c.GroupAdd))
+
+	for i, g := range c.GroupAdd {
+		group, err := strconv.Atoi(g)
```

Comment:
If you are just going to atoi() it, why not make it []int instead of []string in the first place?  Accepting string means validation is more clumsy and it implies that accepting a group name is in the plans